lib/music/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_reader :id, :items
+  attr_accessor :name
+
+  def initialize(name)
+    @id = Random.rand(1...10_000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+  end
```

Comment:
@derrick1451  requested changes on this pull request.

Hi @Benawi , 👋🏻👋🏻
Good job so far! 💯⭐🔥🚀🤩
There are some issues that you still need to work on to go to the next project but you are almost there!
### Highlights
- No Linters error 🤩
- Followed Gitflow 🙏🏻
- Professional README ✨
- Good working on the functionality 🔥
- ### Required changes 
- should add self as a property of the item object (by using the correct setter from the item object)
for example item.genre=self